scipy/sparse/linalg/tests/test_interface.py

```diff
@@ -284,6 +284,20 @@ def always_four_ones(x):
         assert_(hasattr(op, "shape"))
         assert_(hasattr(op, "_matvec"))
 
+def matvec(x):
+    """ Needed for test_pickle as local functions are not pickleable """
```

Comment:
Minor: better not add docstrings for test functions/methods (they clutter the verbose output of `nosetests`/'runtests.py'). Would be better to remove it if/when you're squashing commits.